nototools/unicode_data.py

```diff
@@ -688,6 +693,117 @@ def _load_emoji_data():
         emoji_sets['Emoji'])
 
 
+def _load_emoji_sequence_data():
+    """Parse emoji-sequences.txt"""
+    global _emoji_combining_sequences, _emoji_flag_sequences
+    global _emoji_modifier_sequences
+    if _emoji_combining_sequences:
+        return
+    emoji_maps = {
+        'Emoji_Combining_Sequence': {},
+        'Emoji_Flag_Sequence': {},
+        'Emoji_Modifier_Sequence': {},
+    }
+    map_names = '|'.join(sorted(emoji_maps.keys()))
+    line_re = re.compile(
+        r'([0-9A-F ]+);\s+(%s)\s+#\s+(\d+\.\d+)\s.*?(?:Flag for (.*))?$' %
+        map_names)
+    with open_unicode_data_file('emoji-sequences.txt') as f:
+        for line in f:
+            line = line.strip()
+            if not line or line[0] == '#':
+                continue
+            m = line_re.match(line)
+            if not m:
+                raise ValueError('Did not match "%s"' % line)
+            seq = tuple(int(s, 16) for s in m.group(1).split())
+            emoji_map = emoji_maps[m.group(2)]
+            is_flag = m.group(2) == 'Emoji_Flag_Sequence'
+            age = float(m.group(3))
+            flag_name = m.group(4)
+            value = (age, flag_name) if is_flag else age
+            emoji_map[seq] = value
+    _emoji_combining_sequences = emoji_maps['Emoji_Combining_Sequence']
+    _emoji_flag_sequences = emoji_maps['Emoji_Flag_Sequence']
+    _emoji_modifier_sequences = emoji_maps['Emoji_Modifier_Sequence']
+
+
+def _load_emoji_zwj_sequence_data():
+    """Parse emoji-zwj-sequences.txt."""
+    global _emoji_zwj_sequences
+    if _emoji_zwj_sequences:
+        return
+    line_re = re.compile(
+        r'([0-9A-F ]+);\s+Emoji_ZWJ_Sequence\s+'
+        r'#\s(\d+.\d+)\s.*?\)\s+(?:([^:]+):|EYE, LEFT SPEECH BUBBLE).*$')
+    data = {}
+    with open_unicode_data_file('emoji-zwj-sequences.txt') as f:
+        for line in f:
+            line = line.strip()
+            if not line or line[0] == '#':
+                continue
+            m = line_re.match(line)
+            if not m:
+                raise ValueError('Did not match "%s"' % line)
+            seq = tuple(int(s, 16) for s in m.group(1).split())
+            age = float(m.group(2))
+            seq_name = m.group(3)
+            # patch in preferred name for this anomalous value
+            if seq_name == None:
```

Comment:
Style guide prefers comparing agains None with `is` instead of `==` (not sure why though)
http://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations,
also applies to two later comparisons a few lines down.